config/crd/bases/toolchain.dev.openshift.com_bannedusers.yaml

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
is this change intentional or arising from a formatting difference? many other files also have similar changes (removing a blank line), so just wanted to confirm

Reply:
yes, yamllint was complaining about too many empty lines